src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,84 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
```

Comment:
Please refine your short summary and long summary to make it more descriptive and start with Capital letter. At least you should keep your help message similar with `az reservations reservation-order list` command.

Reply:
"Calculate price" is not very clear for users to understand what your command do.